### ISSUE TITLE
[BUG] Update to streamlink 6.0.0-1 breaks wtwitch

### ISSUE THREAD

**Issue by 3ship** [krathalan/wtwitch#24]
**Describe the bug**
Since just now I can no longer open streams via `wtwitch`. With every streamer in my follow list I get an error message saying that the stream doesn't support the configured quality. I had the quality set to `best` but the same happens with every other configuration I try.

Just before this error started appearing I updated my system and `streamlink` was updated from version 5.5.1-1 to 6.0.0-1.

I can watch Twitch streams in `mpv` by executing either `streamlink -p mpv twitch.tv/forsen best` or `mpv https://twitch.tv/forsen`

**To Reproduce**
`wtwitch w forsen`

**Expected behavior**
Stream opens as usual in mpv.

**Logs/system info (please complete the following information):**
 - OS: EndeavourOS
 - `wtwitch` version: 2.6.2

The likely relevant portion from the debug log:

```
[cli][info] Opening stream: 1080p60 (hls)
[cli][info] Starting player: mpv --hwdec=auto --gpu-context=wayland
[plugins.twitch][info] Will skip ad segments
[plugins.twitch][info] Waiting for pre-roll ads to finish, be patient
[stream.hls][info] Filtering out segments and pausing stream output
[stream.hls][warning] Encountered a stream discontinuity. This is unsupported and will result in incoherent output data.
[stream.hls][info] Resuming stream output
error: Failed to start player: mpv --hwdec=auto --gpu-context=wayland (Player executable not found)
[cli][info] Closing currently open stream...
+ stream_failure_notify forsen 'forsen'\''s stream does not support quality best.'
+ local localizedStrings
+ mapfile -t localizedStrings
++ jq -r '.stream_failure_one.en, .stream_failure_two.en'
+ notify-send -i gnome-twitch 'Wtwitch: watching forsen failed' 'forsen'\''s stream does not support quality best.'
+ exit_script_on_failure
```

**Comment by 3ship** [comment]
Just to confirm: Downgrading `streamlink` to the previous version makes `wtwitch` work again.

**Comment by mdxio** [comment]
Streamlink 6.0.0 changed the `--player`  argument into a path-only-argument and added a `--player-args` argument.

https://github.com/streamlink/streamlink/releases/tag/6.0.0

So now streams won't open with mpv because when wtwitch calls `streamlink` with mpv as the default player, the options  `--hwdec=auto` (and `--gpu-context=wayland` if using wayland) are added to the `--player` argument which should only contain a path to mpv or just the name itself. 

Streams will open fine with vlc though and that's because no arguments are being added to the `--player` argument when vlc. is the default player

**Comment by krathalan** [comment]
Fixed with PR#25, thanks
